spec/unit/mutant/cli_spec.rb

```diff
@@ -237,7 +237,13 @@
       let(:expected_matcher_config) do
         default_matcher_config.with(
           subject_filters: [
-            Mutant::Repository::SubjectFilter.new(Mutant::Repository::Diff.new('HEAD', 'master'))
+            Mutant::Repository::SubjectFilter.new(
+              Mutant::Repository::Diff.new(
+                config: Mutant::Config::DEFAULT,
+                from: 'HEAD',
```

Comment:
I've to check rubocop, they ship some cops that are disabled by default, maybe our case is covered.